metadata/metadata_tutorial_addition.Rmd

```diff
@@ -0,0 +1,622 @@
+
+# Section 2: Required input
+
+URL to a public published google sheet (you will need to provide this url below in section 4)
+
+   * copy and fill out the following [google sheets template](https://docs.google.com/spreadsheets/d/1pNEuwNHNHHrpql0b36DYFAbOlBU5TJVF_nO4RdRzdmg/edit#gid=279516453).
+   * instructions on how to fill out the template can be found under the README worksheet.
+   * make the google sheet public (to anyone with link) and publish the entire document to the web as a xlsx.
+
+# Section 3: How to Run this Rmd
+
+## Commit database transactions to upload data
+
+Knitting this markdown file will not upload any data to the test database. Being able to knit this file means that you will not encounter any errors when uploading your metadata. All experiments, sites, treatments, cutlivars, and citations can be uploaded successfully. 
+
+To actually commit additions to the test database, you will need to make an edit to this markdown file. 
+
+The value of `commit` needs to be changed from `FALSE` to `TRUE` in the following chunk:
+
+```{r commit-param}
+
+commit <- FALSE
+
+```
+
+It is important to note that you should only be uploading your metadata once. 
+
+Trying to upload your metadata more than once will result in errors. 
+
+If you would like to reinitialize the test database and remove any additions that you have made, re-run step 3 of the `How to set up local instance of BETY test database` tutorial.
+
+## Check uploaded data
+
+You will not be able to fetch any of your uploaded data until it has been added to the database. 
+
+You can check your uploaded results when you run this Rmd with `commit <- TRUE` in the above chunk. 
+
+The value of `eval_chk` will depend on the value you set for `commit`. This variable will determine whether chunks for checking uploaded data will be evaluated. You will not see any tables below these chunks unless you set commit to TRUE. 
+
+```{r check-upload, results = 'hide'}
+
+ifelse(commit == TRUE, eval_chk <- TRUE, eval_chk <- FALSE)
+
+```
+
+# Section 4: Metadata upload
+
+## Set up
+
+Please provide the URL to your google sheet below. 
+
+A sample URL has been provided for a google sheet containing MAC season 8 metadata. Replace the sample URL with your own.
+
+```{r url}
+
+url <- 'https://docs.google.com/spreadsheets/d/1BPkZPfynZI9DB9bRJfa335uC_uNjST5jHVf-XUgSVvM/edit#gid=347205052' 
+
+```
+
+The following packages will need to be loaded: `RPostgres`, `readxl`, `dplyr`, `googlesheets`, `kableExtra`, `glue`, and `DBI`
+
+```{r load-pack, message = FALSE}
+
+library(RPostgres)
+library(readxl)
+library(dplyr)
+library(googlesheets)
+library(kableExtra)
+library(glue)
+library(DBI)
+
+```
+
+A connection to the test database needs to be created. Make sure that you have the test database running. 
+
+The port number should be the same as the one indicated in docker-compose.override.yml.
+
+```{r create-dbcon}
```

Comment:
I would move this to the section "Upload data", i.e. just before it is needed; I think that would be more clear.